docs: added a link to the contributing guidelines in the PR template (#1808)
netlify/next-runtime: CONTRIBUTING.md

🎉 Thanks for considering contributing to this project! 🎉

> When contributing to this repository, please first discuss the change you wish to make via an

Author: Feel free to suggest better wording.

[issue](https://github.com/netlify/next-runtime/issues/new/choose). Please use the issue templates. They are there to
help you and to help the maintainers gather information.

Before working on an issue, ask to be assigned to it. This makes it clear to other potential contributors that someone
is working on the issue.

> When creating a PR, please fill out the template to the best of your abilities. Do not delete it. The information in the
> template helps maintainers review your pull request.

Reviewer: [sand] not a big deal, but maybe this sounds a bit heavy handed?
```suggestion
When creating a PR, please use the template. The information in the template helps maintainers review your pull request.```

Reviewer 2: +1

Most common commit message prefixes are:

> - `fix:` which represents bug fixes, and generate a patch release.

Author: Looks like prettier in action.
